cli/command/image/build.go

```diff
@@ -132,6 +134,7 @@ func NewBuildCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags.BoolVar(&options.squash, "squash", false, "Squash newly built layers into a single new layer")
 	flags.SetAnnotation("squash", "experimental", nil)
 	flags.SetAnnotation("squash", "version", []string{"1.25"})
+	flags.VarP(&options.volumes, "volume", "v", "Bind mount a volume")
```

Comment:
shouldn't it be in plural? `"Bind mount volumes"`

Reply:
It's the same wording as normal volumes